lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -440,6 +440,9 @@ def check_github_response
       fail_with_message(e, "Exceeded retry limit")
     rescue Octokit::Unauthorized => e
       fail_with_message(e, "Error: wrong GitHub token")
+    rescue StandardError => e
+      Helper.log.error("#{e.class}: #{e.message}")
+      nil
```

Comment:
🤔 This part "skip all other errors" does not seem to be optional, and the `continue_with_errors` setting is mostly "read" at the beginning, then no more is done with it.

Is that intentional?